modules/core/src/test/resources/org.apache.ignite.util/GridCommandHandlerClusterByClassTest_help.output

```diff
@@ -27,7 +27,10 @@ This utility can do the following commands:
       ACTIVE_READ_ONLY  - Activate cluster. Cache updates are denied.
 
   Print cluster baseline topology:
-    control.(sh|bat) --baseline
+    control.(sh|bat) --baseline [--verbose]
```

Comment:
I believe we should also update GridCommandHandlerClusterByClassWithSSLTest_help.output